cmd/root/results.go

```diff
@@ -27,55 +27,81 @@ func requiredRunes(s string) []rune {
 	return arr
 }
 
-// Calculates the words per minute based on the calculations in this link:
-// https://www.speedtypingonline.com/typing-equations
-func wpm(start time.Time, end time.Time, typed string, exercise string, wordSize int) float64 {
-	if start.After(end) {
-		end = time.Now()
-	}
-	minLengthString := exercise
-	if len(typed) < len(exercise) {
-		minLengthString = typed
-	}
-	mins := end.Sub(start).Minutes()
-	incorrect := float64(numIncorrectCharacters(typed, exercise))
-	typedEntries := len(requiredRunes(minLengthString))
-	words := float64(typedEntries / wordSize)
+func wpm(events []event) float64 {
```

Comment:
I do want to make this one function that returns both the wpm and the raw wpm, since I'm doing a lot of the same stuff in both functions, but that can happen later.